packages/hashpeek/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=hashpeek
+pkgver=0.2.2
```

Comment:
please update version with makepkg -o or [ba-dev](https://gist.github.com/noraj/5045be32490f1161161e16fc5820066c#testing)

Reply:
@ph4mished i'll do it this time what @noraj mentioned. please next time make sure to run a `makepkg -od` before PR